### PR TITLE
Babylon authority economy and migration (& Slavery)

### PR DESCRIPTION
**Pull request by Unintuition-Time**
Contributes to #95. Changes Economic Systems to make them more differentiated on who contributes, adds penalties to socialist laws to reflect the international trade pariah aspect, and added some buffs/debuffs to better reflect each law.

For migration, added some benefits and detriments to each law, with Migration controls being the milquetoast "middle ground" that most nations will shoot for over time.

For slavery, added Pro-Slavery ideology to Industrialists to Debt Slavery to make it a stickier law than it used to be and create a struggle for nations with this law to remove it. It used to have 0 support and was easily removable for context.

@HiddeLekanne @Danarca ready for review

### CODE REVIEW

File: common/laws/00_economic_system.txt [hunk 1]
```diff
@@ -387,57 +349,48 @@ law_industry_banned = {
...
-	
 	on_activate = {
 	}
-	
 	modifier = {
 		country_loan_interest_rate_mult = -0.15
-		state_shopkeepers_investment_pool_efficiency_mult = 0.25	
-		state_capitalists_investment_pool_efficiency_mult = 0.25
+		state_shopkeepers_investment_pool_efficiency_mult = 0.5
+		state_capitalists_investment_pool_efficiency_mult = 0.3
```
Comment (HiddeLekanne) [inline common/laws/00_economic_system.txt]: I don't like buffing this capitalist efficiency. It's already fairly powerful.

File: common/laws/00_economic_system.txt [hunk 2]
```diff
@@ -493,33 +443,27 @@ law_laissez_faire = {
...
 		custom_tooltip = {
 			text = enact_law_radical_leftist_economy_warning_desc
-		}	
+		}
 	}
-	
 	modifier = {
-		state_shopkeepers_investment_pool_efficiency_mult = 0.35	
-		state_farmers_investment_pool_efficiency_mult = 0.35
+		country_law_enactment_time_mult = 0.15
```
Comment (HiddeLekanne) [inline common/laws/00_economic_system.txt:460]: Ideal would be together with council government. But (as I said somewhere else) making conditional combination effects is beyond the scope of here.

File: common/laws/00_economic_system.txt (hunk 1 above)
Comment (Unintuition-Time) [reply common/laws/00_economic_system.txt]: Agreed, will look for alternative change

**Comment by Unintuition-Time** [comment]
Ready for second review @HiddeLekanne

**Review by HiddeLekanne — APPROVED** [review]

**Pull request merged**